presto-main/src/main/java/com/facebook/presto/sql/planner/plan/MarkDistinctNode.java

```diff
@@ -32,6 +33,7 @@
         extends PlanNode
 {
     private final PlanNode source;
+    private final VariableReferenceExpression markerVariable;
     private final Symbol markerSymbol;
```

Comment:
We can just replace all internal Symbol to Variable? We can do a conversion whenever we can getOutput.

Reply:
Ya, try that and tell me how that feels.